data/human/human missions.txt

```diff
@@ -7453,3 +7453,233 @@ mission "Adelita 5"
+			branch meanopening
+				"syndicate business: mean to Howard" > 0
+			`	"Hey <first>,`
+			`		"I hope all is well. I'm excited to send you this interim Megaparsec report! I'm sharing because it's the first one with your investment included--check out Section 3, Part C, Paragraph 6 for the details."`
+				goto openingchoice
+			label meanopening
+			`	"Captain <first> <last>,`
+			`		"Here's the interim Megaparsec report, sent because your investment is included for the first time. You can find the relevant information in Section 3, Part C, Paragraph 6."`
+			label openingchoice
+			branch goodchoice
+				has "Syndicate Business expansion 10m good"
+			branch evilchoice
+				has "Syndicate Business expansion 10m evil"
+			# neutral
+			`		Nothing major to flag, just routine updates. Steady progress and business as usual. Just like I predicted, we're seeing some solid returns that aren't making headlines. And the higherups like the consistency--keeps me in their good graces.`
+				goto howardbye
+			label goodchoice
+			`		"Returns are about what we projected, a little slow, but here's the kicker: we're saving a chunk of provisions for expected environmental lawsuits and employee morale at the expansion is through the roof. Looks like we're building more than just profits. My bosses are thrilled--they think I'm working magic here.`
+				goto howardbye
+			label evilchoice
+			`		"There was a... noteworthy incident involving some employees, so it might draw more attention than we'd like. Nothing that'll derail the returns, though, just some bad press to manage. Everything's still on track, and either way, no one can argue with the numbers."`
+			label howardbye
+			branch alreadyrich
+				"credits" >= 300000000
+			`		"Anyway, here's the mentioned area of that report. I have another investment opportunity coming up, but if my bank contact is correct you have some hustling to do if you want in. Maybe three or four hundred million credits ought to do it. Keep in touch."`
+				goto signoff
+			label alreadyrich
+			`		"Anyway, here's the mentioned area of that report. As long as you stay rich I'll have another investment opportunity for you soon. Keep in touch."`
+			label signoff
+			`			Howard Vendeur`
+			`			Investor Relations Manager, Megaparsec Incorporated`
+			choice
+				`	(Open Section 3, Part C, Paragraph 6 of the attached report.)`
+					goto sectionmarked
+				`	(Take a look at the main report.)`
+				`	(Close out of the message.)`
+					decline
+			# This report has 2 purposes beyond being a peek behind the curtain of Megaparsec's operations
+			# 1. This should be dull and impersonal enough that a reader could easily gloss over Megaparsec's morally questionable actions listed in the report and to be a stark contrast to the human aspect of the other culture mission.
+			# 2. This should be long enough that Megaparsec feels overwhelming in size and scope, and that the player's actions/investments feels like a footnote in comparison.
```

Comment:
```suggestion
			# 2. This should be long enough that Megaparsec feels overwhelming in size and scope, and that the player's actions/investments feels like a footnote in comparison.

```
Improve readability.

Reply:
Could you point to which part is unclear and needs clarification? For comments the most important thing is conveying information clearly above anything prose-related, and from my end I'm not sure where that gap is here.

Reply:
It's somewhat hard to tell where the text switches between comment and actually what the player will see.
On my first read through, I missed the break for a moment.
So I was thinking having an empty line would help with that.

Reply:
Ah I'm blind! Didn't see the extra line so I thought you were asking for phrasing changes improve readability. My bad, an extra line completely makes sense there.

Reply:
Sorry, in retrospect what I had said didn't make much sense.